lib/modules/dosomething/dosomething_rogue/dosomething_rogue.cron.inc

```diff
@@ -56,9 +56,9 @@ function dosomething_rogue_retry_failed_reportbacks() {
       } else {
         $data = [
           [
-            'rogue_reportback_item_id' => $task->rogue_item_id,
+            'rogue_event_id' => $task->rogue_event_id,
             'status' => $task->status,
-            'reviewer' => $task->reviewer,
+            'event_type' => 'post_photo',
```

Comment:
`source` lives on the photo though so I was expecting `something_photo`, I probably missed the decision on this but just wondering because it's not really a post. I totally agree with the signup situation though that part is confusing.